Task_TheGridProblem/index.css

```diff
@@ -1 +1,11 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{

```

Comment:
The 'gap' property is not supported in all browsers, consider using 'grid-row-gap' and 'grid-column-gap' for better compatibility.